rollup.config.js

```diff
@@ -9,7 +8,7 @@ function makeBundle({ filePath, dir = 'dist' }) {
       dir: dir,
       format: 'cjs',
     },
-    plugins: [typescript(), terser()],
```

Comment:
I don't feel like I need to save some kb for a Nodejs app.
Do not terser it helps me debug faster.
I need to learn about source map, to see if it resolves my problem.